coralproject/cay
# update to react 15.0.2

## What does this PR do?

Updates to React 15.x

## How do I test this PR?

- it's probably going to be easiest to nuke `node_modules` and `npm install`.
- `redux-devtools` is now available as a [Chrome plugin](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd). So, I've removed as a dependency in this project.
- make sure the app runs and the redux tools receive actions from our store.

@coralproject/frontend
